stellarphot/settings/models.py

```diff
@@ -410,6 +412,68 @@ def earth_location(self):
         )
 
 
+class PhotometryOptions(BaseModelWithTableRep):
```

Comment:
Are we providing a function for setting the PhotometryOptions or does pydantic do that for us?